### ISSUE TITLE
parser error (missing spaces)

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1412]
Not entirely sure what your stance on these is, do you want to parse everything or should the code not contain these errors as a prerequisite for abaplint? I can fix this up of course.

nevertheless, parser\_missing\_space does not catch this one

```abap
DATA(test) = VALUE string_table(
      ( `( 1 =`)
      ( `=` )
      ( `( )`)
      ( `( ( ) )`)
      ( `1 "A" ABS N000001 ) = 2 "B"`)
      ( `2 3 = 6 5 = AND` )
      ( `,` )
      ( `"A" "B" =` )
      ( `"A" "B" AND` )
      ( `1 = ( 1 , )` )
      ( `1 = ( 1 , , )` )
      ( `1 = ) 1 , , , (` )
      ( `Round 1 , 2 , , , , , ) = 1` )
      ( `1 NOT = 1 AND 1 = 2` )
      ( `1 = 1 AND 1 NOT = 2` )
      ( `1 = 1 NOT AND 1 = 2` ) ).
```

**Comment by larshp** [comment]
@FreHu try again, 2.57.0
